js/help.js

```diff
@@ -0,0 +1,49 @@
+const helpOverlay = new Image();
+helpOverlay.src = 'images/help-image.png';
+
+const helpBtn = document.querySelector('.help');
+
+let helpImage = false;
+
+function drawHelp() {
+  if (helpImage === true) {
+    ctx.drawImage(helpOverlay, 15, 0);
+  }
+}
+
+function introText() {
+  //Intro text
```

Comment:
If you'd like to comment functions, use [JSDoc](https://jsdoc.app/about-getting-started.html). It allows even to generate some documentation out of the code. And type checking with Typescript compiler.

Reply:
And normally in Javascript it is used to comment functions and variables on the line above, not on the right of the variable declaration or inside the function body.